src/components/FormSignUp.vue

```diff
@@ -0,0 +1,165 @@
+				<div>
+					<label for="name" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Name</label
+					>
+					<input
+						v-model="formFields.name"
+						type="text"
+						name="name"
+						id="name"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						placeholder="John Doe"
+						required
+					/>
+				</div>
+				<div>
+					<label for="email" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Your email</label
+					>
+					<input
+						v-model="formFields.email"
+						type="email"
+						name="email"
+						id="email"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						placeholder="name@company.com"
+						required
+					/>
+				</div>
+				<div>
+					<label for="password" class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Your password</label
+					>
+					<input
+						ref="password"
+						v-model="formFields.password"
+						type="password"
+						name="password"
+						id="password"
+						placeholder="••••••••"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						required
+					/>
+				</div>
+				<div>
+					<label
+						for="confirm-password"
+						class="mb-2 block text-sm font-medium text-gray-900 dark:text-white"
+						>Confirm password</label
+					>
+					<input
+						ref="confirmPassword"
+						v-model="formFields.confirmPassword"
+						type="password"
+						name="confirm-password"
+						id="confirm-password"
+						placeholder="••••••••"
+						class="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-gray-900 focus:border-primary-500 focus:ring-primary-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-primary-500 dark:focus:ring-primary-500 sm:text-sm"
+						required
+					/>
+				</div>
+				<div>
+					<button
+						type="submit"
+						class="w-full rounded-lg bg-primary-700 px-5 py-3 text-center text-base font-medium text-white hover:bg-primary-800 focus:ring-4 focus:ring-primary-300 dark:bg-primary-600 dark:hover:bg-primary-700 dark:focus:ring-primary-800 sm:w-auto"
+					>
+						Create account
+					</button>
+					<span v-if="errorMessage" class="mb-4 ml-2 w-full text-xs text-red-600">{{
+						errorMessage
+					}}</span>
```

Comment:
Hahaha, most pretty error message : ) Maybe we could use the alert from [Flowbite](https://flowbite.com/docs/components/alerts/#alerts-with-icon)

```suggestion
					<div
						v-if="errorMessage"
						class="mt-6 flex items-center rounded-lg bg-red-50 p-4 text-sm text-red-800 dark:bg-gray-700 dark:text-red-400"
						role="alert"
					>
						<svg
							class="mr-3 inline h-4 w-4 flex-shrink-0"
							aria-hidden="true"
							xmlns="http://www.w3.org/2000/svg"
							fill="currentColor"
							viewBox="0 0 20 20"
						>
							<path
								d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5ZM9.5 4a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3ZM12 15H8a1 1 0 0 1 0-2h1v-3H8a1 1 0 0 1 0-2h2a1 1 0 0 1 1 1v4h1a1 1 0 0 1 0 2Z"
							/>
						</svg>
						<span class="sr-only">Info</span>
						<div><span class="font-medium">Failure!</span> {{ errorMessage }}</div>
					</div>
```